actionpack/test/dispatch/debug_exceptions_test.rb

```diff
@@ -26,9 +26,9 @@ def hello
   class Boomer
     attr_accessor :closed
 
-    class NilAnnotedSourceCodeError < StandardError
-      def annoted_source_code
-        nil
+    class NilAnnotatedSourceCodeError < StandardError
+      def annotated_source_code
+        []
```

Comment:
This test was meant to cover what happens when `annotated_source_code` returns nil. Changing this method to return an empty array means the test still doesn't cover that case.

Reply:
@eugeneius  Currently `TypeError: no implicit conversion of String into Array` error is raised when `annotated_source_code ` is nil. I have updated the code, Please have a check on it. Thanks for reviewing the changes.